Corona-Warn-App/src/test/java/de/rki/coronawarnapp/contactdiary/ui/overview/ContactDiaryOverviewViewModelTest.kt

```diff
@@ -69,6 +69,7 @@ open class ContactDiaryOverviewViewModelTest {
         every { taskController.submit(any()) } just runs
         every { contactDiaryRepository.locationVisits } returns flowOf(emptyList())
         every { contactDiaryRepository.personEncounters } returns flowOf(emptyList())
+        every { contactDiaryRepository.testResults } returns flowOf(emptyList())
```

Comment:
should we consider Screenshots tests here or we have another task for that ?

Reply:
there is a separated subtask for this so it will be in another PR